qemu/tests/cfg/nic_acpi_index.cfg

```diff
@@ -0,0 +1,26 @@
+- nic_acpi_index:
+    virt_test_type = qemu
+    type = nic_acpi_index
+    only Linux
+    no q35
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    nic_hotplug_count=1
+    pci_model = virtio-net-pci
+    required_qemu = [6.0.0,)
+    kernel_extra_params_remove = biosdevname=0 net.ifnames=0
+    kernel_extra_params_serial_login = yes
+    variants:
+        - with_acpi:
+            vhost = on
+            nic_extra_params += ",acpi-index=1"
+            type = nic_acpi_index_boot
+        - hotplug_same_slot:
+            nics = ""
+            nic_extra_params_hotplug_nic1 = ",acpi-index=2,addr=0x5"
```

Comment:
The addr slot should be generated by vt, but if you want to assign a fixed slot, you can use `nic_pci_addr = 0x5` I think.

Reply:
Hi @PaulYuuu Due to the limitation of the current code, I cannot use this parameter at the moment. So keep the current usage.